src/components/ItemsList.js

```diff
@@ -104,34 +105,31 @@ const ItemsList = () => {
   const listHasNoItems = shoppingList && !shoppingList.length;
 
   return (
-    <div className="max-h-screen flex flex-col box-border items-center">
-      <header className="bg-green-400 w-full fixed text-center">
-        <h2 className="pt-6 pb-16 text-4xl font-thin text-gray-100">
-          Your Shopping List
-        </h2>
-        <span className="text-white top-0 right-0 absolute sm:mt-4 sm:mr-4">
+    <div className="bg-gradient-to-b from-green-300 to-blue-700 max-h-screen box-border flex flex-col items-center font-sans">
+      <header className="w-full fixed text-center text-gray-800">
+        <h2 className="pt-8 pb-16 text-4xl font-thin">Your Shopping List</h2>
+        <span className="absolute top-0 right-0 md:hidden">
           <HomeIcon />
         </span>
       </header>
-      <main className="bg-white relative w-full h-full mt-24 rounded-t-3xl overflow-auto">
+      <main className="bg-white relative w-full h-screen mt-28 pb-24 md:pb-0 rounded-t-3xl shadow-top overflow-auto">
         {loading && (
           <img className="m-auto w-12" src={spinner} alt="Loading..." />
         )}
         {error && <p>An error has occured...</p>}
         {listHasNoItems && (
-          <div className="h-64 bg-white flex flex-col w-screen justify-center items-center text-gray-900">
+          <div className="h-64 flex flex-col w-screen justify-center items-center text-gray-900">
             <p className="">You haven't created a shopping list yet...</p>
-            <button
-              className="bg-white-100 px-6 py-3 text-sm mt-6 border border-color-gray-500 border-solid rounded shadow-md hover:bg-green-500 cursor-pointer hover:text-white"
-              type="submit"
-              onClick={handleRedirect}
-            >
+            <button className="add-btn" type="submit" onClick={handleRedirect}>
               Add First Item
             </button>
           </div>
         )}
         {listHasAtLeastOneItem && (
-          <div className="mt-6 max-w-md mx-auto overflow-auto">
+          <section className="pt-6 max-w-md mx-auto overflow-auto">
+            <nav>
+              <AddItemButton />
```

Comment:
I don't know if having the button at the top is the best option. I kind of liked it better when it was at the bottom😫😫
What do other reviewers think.

Reply:
This was at least the easiest way to solve the overlay problem.. I also do not agree 100% with this solution because of optical reasons but would go for it for now to finish the app until Sunday.

Reply:
To me it even looks better at the top than at the bottom

Reply:
I quite like them at at the top too - and I think maybe the UX is better as they are more visible there?

Reply:
That's totally it. I can also imagine that maybe how they are styled could make them seem a bit out of place for someone.  (color, esp. !!shadow!!..). Especially for the list view, and when the list would be long, the button would seem lost on the bottom. This really worked well for the add items view. But styling them differently I would not suggest. We will see how they look when changing some color today. I also have this "OMG" in the beginning, but looking at some changes an hour later it feels like "oh, actually not that bad".  One suggestion I could think of since we now have 2 buttons on both sides is to put them in the middle left and right.